File: number-of-good-pairs/number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        good_pairs = 0
        
        for i in range(len(nums)):
            start_point = i+1
            for j in range(start_point, len(nums)):
                if nums[i] == nums[j]:
                    good_pairs +=1
        return good_pairs

File: number-of-good-pairs/number-of-good-pairs.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var numIdenticalPairs = function(nums) {
    let good_pairs = 0;
    for(let i=0; i<nums.length; i++){
        for (let j=i+1; j<nums.length; j++){
            if(nums[i] == nums[j]){
               good_pairs += 1
                console.log(good_pairs)
            };
        };
    };
    return good_pairs;
};